data/tools/wmllint

```diff
@@ -835,10 +835,10 @@ linechanges = (
         
         # renamed khalifate to dunefolk
         ("id=khalifate", "id=dunefolk"),
```

Comment:
Shouldn't this be `race=` rather than `id=`?

Reply:
I do have `race=` a few lines below.